src/index.tsx

```diff
@@ -9,8 +13,17 @@ import { Container } from "./container";
 export const Pane: PaneDefinition = {
   icon: `${icons.iconBase}markdown.svg`,
   name: "markdown file",
-  label: (subject) =>
-    subject.uri.endsWith(".md") ? "Handle markdown file" : null,
+
+  label: function (
+    subject: NamedNode,
+    context: DataBrowserContext
+  ): string | null {
+    const t = context.session.store.findTypeURIs(subject);
+    if (t["http://www.w3.org/ns/iana/media-types/text/markdown#Resource"]) {
+      return "Handle markdown file";
```

Comment:
The label is for the user, not the developer. It is used for a hover tool tip on the pane button icon. So "Markdown" instead of "handle markdown file"